Samples/Client/DirectxWin32/src/main.cpp

```diff
@@ -27,30 +28,41 @@
 #pragma comment(lib, "winmm.lib")
 
 //--------------------------------------------------------------------------------------
-// Global Methods
+// Global vars
 //--------------------------------------------------------------------------------------
-std::string GetAbsolutePath(std::string fileName)
-{
-	TCHAR buffer[MAX_PATH];
-	GetModuleFileName(NULL, buffer, MAX_PATH);
-	char charPath[MAX_PATH];
-	wcstombs(charPath, buffer, wcslen(buffer) + 1);
+using namespace StreamingToolkit;
 
-	std::string::size_type pos = std::string(charPath).find_last_of("\\/");
-	return std::string(charPath).substr(0, pos + 1) + fileName;
-}
+WebRTCConfig	g_webrtcConfig;
```

Comment:
can we scope this to `InitWebRTC` instead?

Reply:
I added *InitWebRTC* method before for the convenience of having both normal D3D app and webrtc streaming app in the same file. Since there is only the webrtc part now, it's very clear and understandable to follow the login in the main method. If you have a better idea to organize it, feel free to go ahead and refactor it.

Reply:
oh, okay, that makes sense. i was more questioning the need for a global here, but i understand why you've done it this way now.